Gini-iOS-SDK/GINIAPIManager.h

```diff
@@ -247,6 +259,7 @@ typedef NS_ENUM(NSUInteger, GiniAPIResponseType){
  * @param cancellationToken     Cancellation token used to cancel the current task.
  *
  * @returns                     A`BFTask*` that will resolve to a NSString containing the created document's ID.
+ * @note                        Only available in default API.
```

Comment:
The first thing this evoked in my mind is the interface segregation principle. It would be cleaner to have a separate document task manager for the accounting api. It would also duplicate code or introduce more overhead to not duplicate code.

I am fine with this solution and I would like to have an assert for verifying that the default API is used. It should throw an exception with a helpful message otherwise. To make sure clients notice that this is only usable with the default API.

Reply:
The added a check for this case but I forgot to added it for the partial one, in the Document Task Manager. Now fixed 👍